pymongo/mongo_client.py

```diff
@@ -1751,9 +1751,7 @@ def _process_periodic_tasks(self) -> None:
                 helpers._handle_exception()
 
     def __start_session(self, implicit: bool, **kwargs: Any) -> ClientSession:
-        # Raises ConfigurationError if sessions are not supported.
         if implicit:
-            self._topology._check_implicit_session_support()
             server_session: Union[_EmptyServerSession, _ServerSession] = _EmptyServerSession()
         else:
             server_session = self._get_server_session()
```

Comment:
Let's have both implicit and explicit sessions start with _EmptyServerSession(). That way we can check logicalSessionTimeoutMinutes on first use in _materialize().